src/clj/witan/send/params.clj

```diff
@@ -80,11 +82,16 @@
                                    (assoc coll ay {:alpha (double (/ alpha total))
                                                    :beta (double (/ beta total))})))
                                {} observations)
-        prior-per-year (continue-for-latter-ays prior-per-year academic-years)]
+        unobserved-priors (reduce (fn [coll ay]
```

Comment:
`beta-params-leavers` is one of the functions where I struggle to know what each step of the let does. It would be great to factor out each step of the let into their own functions and have at least some simple tests on this to show what we expect each step to look like.

Reply:
Would love to, but pre-existing work depends on this at present.  Will gladly pick up after other fixes are merged.